lib/util/utils.ts

```diff
@@ -184,6 +201,7 @@ export interface ServiceCallback<TResult> {
  * Converts a Promise to a callback.
  * @param {Promise<any>} promise The Promise to be converted to a callback
  * @returns {Function} A function that takes the callback (cb: Function): void
+ * @deprecated generated code should instead depend on responseToBody
```

Comment:
Holding back on actual removal because I know ms-rest-nodeauth and possibly others depend on this function